### PR TITLE
Quarkus insights 63 blog post

### PR DESCRIPTION
**Pull request by kjq**
Quarkus Insights #63 blog post

### CODE REVIEW

File: _posts/2021-09-20-logicdrop-customer-story.adoc [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+* Mongo and Panache did just what we needed nothing more
+* Kubernetes configurations could be kept closer to the service
+* Mutiny made writing reactive services way easier for everyone
+* No more reflection to scan "beans" to help configuration or DI
+* Security worked out-of-the-box and was well defined (big win)
+* REST clients helped isolate services cleaner and better
+* OpenAPI and Swagger worked without tweaking
+* Intimate knowledge of Maven was no longer required
+* Testing became easier and consistent
+* CI/CD was responsible across all services rather than individually
```
Comment (maxandersen) [inline _posts/2021-09-20-logicdrop-customer-story.adoc:94]: Responsible ? Do you mean reproducible or repeatable ? Feels like a word missing ?

Comment (maxandersen) [inline _posts/2021-09-20-logicdrop-customer-story.adoc:94]: Responsible ? Do you mean reproducible or repeatable ? Feels like a word missing ?

**Review by maxandersen — CHANGES_REQUESTED** [review]
Looks great! 

Just a few cleanup changes then we can publish it asap.

File: _posts/2021-09-20-logicdrop-customer-story.adoc [hunk 2]
```diff
@@ -0,0 +1,193 @@
+---
+layout: post
+title: 'How Quarkus Improved Logicdrop's Developer Experience and Process'
```
Comment (maxandersen) [inline _posts/2021-09-20-logicdrop-customer-story.adoc:3]:
```suggestion
title: "How Quarkus Improved Logicdrop's Developer Experience and Process"
```

File: _posts/2021-09-20-logicdrop-customer-story.adoc [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+date: 2021-09-20
+tags: user-story
+synopsis: Moving to Quarkus from a large SpringBoot ecosystem seemed overwhelming at first, but in the end, it was a simple straight-forward transition. Quarkus not only made Java development fun again but also drastically improved the developer experience and Logicdrop's ability to deliver features faster.
```
Comment (maxandersen) [inline _posts/2021-09-20-logicdrop-customer-story.adoc:6]:
```suggestion
synopsis: "Moving to Quarkus from a large SpringBoot ecosystem seemed overwhelming at first, but in the end, it was a simple straight-forward transition. Quarkus not only made Java development fun again but also drastically improved the developer experience and Logicdrop's ability to deliver features faster."
```

File: _posts/2021-09-20-logicdrop-customer-story.adoc [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+* Mongo and Panache did just what we needed nothing more
+* Kubernetes configurations could be kept closer to the service
+* Mutiny made writing reactive services way easier for everyone
+* No more reflection to scan "beans" to help configuration or DI
+* Security worked out-of-the-box and was well defined (big win)
+* REST clients helped isolate services cleaner and better
+* OpenAPI and Swagger worked without tweaking
+* Intimate knowledge of Maven was no longer required
+* Testing became easier and consistent
+* CI/CD orchestrated all of the services rather each one than individually
```
Comment (maxandersen) [inline _posts/2021-09-20-logicdrop-customer-story.adoc:94]:
```suggestion
* CI/CD orchestrated all of the services rather than each one individually
```

**Review by maxandersen — CHANGES_REQUESTED** [review]
turns out the final rendering fails as the frontmatter doesn't have the right quotes.

File: _posts/2021-09-20-logicdrop-customer-story.adoc (hunk 2 above)
Comment (kjq) [reply _posts/2021-09-20-logicdrop-customer-story.adoc:3]: Fixed.  As a note, i copied the previous article here to use as a template.

File: _posts/2021-09-20-logicdrop-customer-story.adoc (hunk 3 above)
Comment (kjq) [reply _posts/2021-09-20-logicdrop-customer-story.adoc:6]: Fixed

File: _posts/2021-09-20-logicdrop-customer-story.adoc (hunk 4 above)
Comment (kjq) [reply _posts/2021-09-20-logicdrop-customer-story.adoc:94]: Fixed.  Made the change too fast without double-checking, my apologies.

File: _posts/2021-09-20-logicdrop-customer-story.adoc (hunk 2 above)
Comment (kjq) [reply _posts/2021-09-20-logicdrop-customer-story.adoc]: Fixed

File: _posts/2021-09-20-logicdrop-customer-story.adoc (hunk 3 above)
Comment (kjq) [reply _posts/2021-09-20-logicdrop-customer-story.adoc]: Fixed

**Comment by kjq** [comment]
All changes made.

**Comment by maxandersen** [comment]
perfect! i'll merge it and will go live on monday

**Pull request merged**